### PR TITLE
Svea icons

### PR DESCRIPTION
**Pull request by mgpkauko**
** Checklist **
- [x ] All icons have a corresponding entry in `db/payment_icons.yml`
- [x ] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x ] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x ] I am confident that all icons are clear and easy to read/understand
- [x ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x ] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by mgpkauko** [comment]
Updated the image files and restored the tags removed by svgo (regardless of disable -attribute). Should be ok now.

File: db/payment_icons.yml [hunk 1]
```diff
@@ -1157,4 +1157,40 @@
...
+  label: Svea Yrityslasku
+  group: other
+
+  name: sveab2binvoice
+  label: Svea B2B Invoice
+  group: other
+  
+  name: sveab2bfaktura
+  label: Svea B2B Faktura
   group: other
```
Comment (larouxn) [inline db/payment_icons.yml:1196]: Looks like these entries are missing the preceding `-`. Please see other entries for reference. (or see below)

https://github.com/activemerchant/payment_icons/blob/b2ebb1241bc8bb7da0d2a196c7576696d7a0bfe4/db/payment_icons.yml#L1-L4

**Review by larouxn — CHANGES_REQUESTED** [review]

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩 
image

(part payment could probably be adjusted a bit margin wise, but it's more or less alright)

**Review by tauthomas01 — APPROVED** [review]

**Comment by larouxn** [comment]
👋 Hello, your icon(s) should be available everyone now. Please let us know if otherwise.

**Pull request merged**